.github/workflows/testing.yml

```diff
@@ -0,0 +1,81 @@
+        numpy-version: ['--upgrade numpy']
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Setup openblas
+      run: |
+        # this setup is originally taken from the .travis.yml
+
+        # virtualenv needed for the multibuild steps
+        pip install virtualenv
+        brew install libmpc gcc@6
+        # The openblas binary used here was built using a gfortran older than 7,
+        # so it needs the older abi libgfortran.
+        export FC=gfortran-6
+        export CC=gcc-6
+        export CXX=g++-6
+        mkdir gcc_aliases
+        pushd gcc_aliases
+        ln -s `which gcc-6` gcc
+        ln -s `which g++-6` g++
+        ln -s `which gfortran-6` gfortran
+        # make gcc aliases in current dir
+        export PATH=$PWD/gcc_aliases:$PATH
+        popd
+        touch config.sh
+        git clone --depth=1 https://github.com/matthew-brett/multibuild.git
+
+        # designed for travis, but probably work on github actions
+        source multibuild/common_utils.sh
+        source multibuild/travis_steps.sh
+        before_install
+        export CFLAGS="-arch x86_64"
+        export CXXFLAGS="-arch x86_64"
+        printenv
+        wget https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/openblas-v0.3.7-macosx_10_9_x86_64-gf_1becaaa.tar.gz -O openblas.tar.gz
+        mkdir openblas
+        tar -xzf openblas.tar.gz -C openblas
+        # Modify the openblas dylib so it can be used in its current location
+        # Also make it use the current install location for libgfortran, libquadmath, and libgcc_s.
+        pushd openblas/usr/local/lib
+        install_name_tool -id $PWD/libopenblasp-r0.3.7.dylib libopenblas.dylib
+        install_name_tool -change /usr/local/gfortran/lib/libgfortran.3.dylib `$FC -v 2>&1 | perl -nle 'print $1 if m{--libdir=([^\s]+)}'`/libgfortran.3.dylib libopenblas.dylib
+        install_name_tool -change /usr/local/gfortran/lib/libquadmath.0.dylib `$FC -v 2>&1 | perl -nle 'print $1 if m{--libdir=([^\s]+)}'`/libquadmath.0.dylib libopenblas.dylib
+        install_name_tool -change /usr/local/gfortran/lib/libgcc_s.1.dylib `$FC -v 2>&1 | perl -nle 'print $1 if m{--libdir=([^\s]+)}'`/libgcc_s.1.dylib libopenblas.dylib
+        popd
+        echo "[openblas]" > site.cfg
+        echo "libraries = openblas" >> site.cfg
+        echo "library_dirs = $PWD/openblas/usr/local/lib" >> site.cfg
+        echo "include_dirs = $PWD/openblas/usr/local/include" >> site.cfg
+        echo "runtime_library_dirs = $PWD/openblas/usr/local/lib" >> site.cfg
+        # remove a spurious gcc/gfortran toolchain install
+        rm -rf /usr/local/Cellar/gcc/9.2.0_2
+
+    - name: Install packages
+      run: |
+        pip install setuptools wheel cython gmpy2 pytest pytest-xdist pybind11 pytest-xdist mpmath
```

Comment:
can we add `scikit-umfpack scikit-sparse`? I think in `.travisci.yml` it's a mistake to only have those in the `'full'` run.

Reply:
It's not [liking the pip install](https://github.com/andyfaff/scipy/runs/573524735?check_suite_focus=true) of scikit-sparse

Reply:
Solved by `brew install suitesparse`